OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -1,9 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Condition="'$(Mono)' != ''">
+    <DefineConstants>MONO</DefineConstants>
+  </PropertyGroup>
```

Comment:
Why is this here all of a sudden?

Reply:
This variable is used the disable/enable certain paths on mono, but was never defined, and nobody noticed that before...

Reply:
Removed it.